tests/unit/zhmcclient_mock/test_urihandler.py

```diff
@@ -4668,6 +5766,7 @@ def test_invoke_ok(self):
         vswitch1['connected-vnic-uris'] = connected_nic_uris
 
         # the function to be tested:
+        # XXX: Fix this to be get instead of post, also in handler itself.
```

Comment:
And what does XXX stand for? :)

Reply:
Same as TODO and FIXME - it is reported by Pylint as a todo. I tend to use XXX for temporary TODOs that I want to address quickly, but in this case apparently I did not. :-(